src/internal/proc.js

```diff
@@ -1,4 +1,8 @@
-import { noop, kTrue, is, log as _log, check, deferred, uid as nextEffectId, array, remove, object, TASK, CANCEL, SELF_CANCELLATION, makeIterator, createSetContextWarning, deprecate, updateIncentive } from './utils'
+import {
```

Comment:
hm, thats becoming quite a long list, but newline shouldnt be inserted here yet, gonna think how those utils could be splitted later